### ISSUE TITLE
Filters always and never not implemented

### ISSUE THREAD

**Issue by Alfalik** [twizmwazin/CardinalPGM#887]
![image](https://cloud.githubusercontent.com/assets/8951324/9978502/1762ab90-5f09-11e5-8a52-7c5030a6a27a.png)

From: https://docs.oc.tc/modules/filters

**Comment by Pablete1234** [comment]
@Alfalik do you know any map that needs this filters to work, but doesn't requiere other missing features on cardinal? (for testing purposes)

**Comment by Alfalik** [comment]
@Pablete1234 I've seen it used on **Sumo**, as shown here:
```xml
<regions>
    <cuboid id="red-glass" min="-171,2,-158" max="-168,5,-155" />
    <cuboid id="blue-glass" min="-166,2,-153" max="-163,5,-149" />
    <apply block="never"/>
</regions>
```
Or at **Desert Sanctuary** (Ranked):
```xml
<regions>
    <apply block="never" region="everywhere"/>
</regions>
```
I've also seen it used in other ways, but I can't remember the exact map. I'll tell you if I find it anyway.

**Comment by Pablete1234** [comment]
@Alfalik i just realized the always and never filters are just the same as "allow-all" or "deny-all" filters, they just changed name in proto 1.4
if you can do some testing in #941 it would be great

**Comment by Alfalik** [comment]
Already implemented, de7dbef.

**Comment by Alfalik** [comment]
`Reopen this as it seems that <apply block="never"/>` is not working (#970).

**Comment by twizmwazin** [comment]
Is this still an issue?

**Comment by Lmmb74** [comment]
The module filter actually applies them, so... I don't think it is still an issue.

**Comment by Pablete1234** [comment]
@twizmwazin I have no idea, i implemented this in de7dbef and that code seems untouched, i don't really know under what circumstances it doesn't work / seems not to be implemented

**Comment by twizmwazin** [comment]
I'm going to close it as we haven't had any recent mentions of it not working/
